docs/src/main/asciidoc/native-and-ssl.adoc

```diff
@@ -5,7 +5,7 @@ https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 ////
 = Using SSL With Native Executables
 include::_attributes.adoc[]
-:categories: core
+:categories: core, native
```

Comment:
Not sure if it makes sense go keep them in `core` if we have a specific section.
I would keep the introduction in `getting-started` though.

But that's probably something you need to discuss with @ebullient and @michelle-purcell .

Reply:
Would Quarkus developers also look for this information under security also? If so, for findability, it might be no harm to add 'security' also.

Reply:
categories do apply to the filter. core, native, security make sense (as this is a core function that doesn't require an extension, right?)

Reply:
Right

Reply:
I added security as well